samples/subsys/usb/audio/src/main.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Sample app for Audio class
+ *
+ */
+
+#include <zephyr.h>
+#include <logging/log.h>
+#include <usb/usb_device.h>
+#include <usb/class/usb_audio.h>
+
+#define LOG_LEVEL 3
+LOG_MODULE_REGISTER(main);
+
+/* This must be global to be able to send data in receive callback */
+struct device *mic_dev;
+
+static int data_received(u8_t ep, struct net_buf *buffer, size_t size)
+{
+	int ret;
+
+	if (!buffer) {
+		/* This should never happen */
+		LOG_ERR("Buffer was not allocated");
+		return -EIO;
+	}
+	if (!size) {
+		LOG_ERR("%d", __LINE__);
```

Comment:
I think we can come up with better log.

Reply:
Yea, for sure :) Forgot to remove it after debugging.